ruby/app.rb

```diff
@@ -4,6 +4,14 @@
 
 set :public_folder, File.dirname(__FILE__) + '/public'
 
+if ENV['PLAID_CLIENT_ID'] == nil
+  raise 'Missing `PLAID_CLIENT_ID`, find your client ID and other API keys at https://dashboard.plaid.com/account/keys'
```

Comment:
same nit as above: can we name it `client_id` instead of client ID, per `public_key` below?